nucypher/blockchain/eth/utils.py

```diff
@@ -136,17 +134,17 @@ def rpc_endpoint_health_check(endpoint: str, max_drift_seconds: int = 60) -> boo
     return True  # finally!
 
 
-def get_default_rpc_endpoints() -> Dict[int, List[str]]:
+@memoize
+def get_default_rpc_endpoints(domain) -> Dict[int, List[str]]:
```

Comment:
Let's also have this take the `TACoDomain` enum. You can then use `domain.name` in the method.

Reply:
done